test/DebuggerCommon/bug_507528.js

```diff
@@ -0,0 +1,28 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Endless calls to cause the stackoverflow. This is to validate the bug 507528

```

Comment:
Will leave it as it is as this was for some bug.

---
In reply to: [77886584](https://github.com/Microsoft/ChakraCore/pull/1537#discussion_r77886584) [](ancestors = 77886584)